packages/peregrine/src/hooks/__tests__/useDOMQuery.spec.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
I have tried using `react-test-renderer` but unfortunately, it is not working as enzyme's `mount` which is a full dom render. It is only rendering components and leaving them in the react world but since we are directly manipulating DOM and bypassing react middle layer, the changes are not showing up in the output. For instance, when I did this:

```
act(() => {
  ReactDOM.render(
    // or TestRenderer.create
    <TestValidComponent
      newText={newText}
      newHTML={newHTML}
      newName={newName}
    />,
    container
  );
});
```

The output looked like this:

```
[
  { type: "div", props: { id: "div1" }, children: ["I am Test Div 1"] },
  {
    type: "div",
    props: { id: "div2", name: "oldName" },
    children: ["I am Test Div 2"]
  }
];
```

I was not able to get full dom output from the renderer.